content/en/platform/corda/5.0-beta/corda-cli-reference/cli-commands/database.md

```diff
@@ -14,7 +14,7 @@ This section lists the Corda CLI `database` arguments. You can use these command
 | <div style="width:160px">Argument</div> | Description                                                                        |
 | --------------------------------------- | ---------------------------------------------------------------------------------- |
 | spec                                    | Generates the database schema from Liquibase.                                      |
-| -c, \-\-clearChangeLog                  | Deletes the changelogCSV in the PWD to force generation of the SQL files.          |
+| -c, \-\-clear-change-log                   | Deletes the changelogCSV in the PWD to force generation of the SQL files.          |
```

Comment:
should these | chars be aligned on the same column ?

Reply:
The table will be rendered ok (spaces ignored) but I've reformatted it to improve readability.